randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -81,15 +83,15 @@ Extra - asset_id: 67
 > Pickup (Power Bomb Expansion); Heals? False
   * Layers: default
   * Pickup 13; Category? Minor
-  > Part-Way up Etecoons Shaft
+  > Door to Green Brinstar Main Shaft
       Morph Ball
 
 > Part-Way up Etecoons Shaft; Heals? False
   * Layers: default
   > Door to Green Brinstar Beetom Room
-      All of the following:
-          Morph Ball
-          Morph Ball Bombs or Power Bombs ≥ 2
+      Any of the following:
+          Can Lay Bombs
+          Power Bombs ≥ 2 and Can Lay Power Bombs
   > Door to Etecoon Energy Tank Room
```

Comment:
i cant find the correct line here, but etecoon energy tank room door -> door to green brinstar main shaft seems to be missing requirements.
its also weird taht its WJ beginner, considering its a similar height as the dachora one which is WJ advanced

Reply:
![image](https://github.com/randovania/randovania/assets/46821362/f2d032fb-4bda-494a-bfac-534b3d7bbb90)

There are requirements. And I would mark them intermediate at best; but to match the other I'll make this one advanced.

Reply:
When I said "missing requirements" i meant that stuff like IBJ and probably more are missing :P

Reply:
Also you may want to have a connection from bottom door -> part way up, as that one generally is easier than going all the way up.

Reply:
Changed the connections.
Doesn't it just seem wrong to have the wall jump tutorial listed as "Advanced" Wall Jumps?

Reply:
Added description of Wall Jump levels to spreadsheet and adjusted the requirements for WJ tutorial and Shinespark tutorial to fit.